examples/with-sass/README.md

```diff
@@ -0,0 +1,55 @@
+# Welcome to Remix!
```

Comment:
Could you update this to use the same README format as in the template?

Reply:
yep sorry, made this before we had a template 😅